runtime/cudaq/domains/chemistry/CMakeLists.txt

```diff
@@ -8,9 +8,17 @@
 
 add_library(cudaq-chemistry SHARED molecule.cpp)
 target_link_libraries(cudaq-chemistry PRIVATE cudaq-spin)
-target_include_directories(cudaq-chemistry 
```

Comment:
It seems to me that this change effectively disables/ignores all warnings for the cudaq-chemistry library. What was you reason for not disabling warnings that are benign on a case-by-case basis instead?

Reply:
All the warnings came from these third-party libraries. None of the warnings came from our code.

Reply:
But as we add code, we may get warnings that are from our code. Why not disable the warnings when including the third party libraries?

Reply:
Oh, I was reading this line wrong - this setting only applies to the headers in tpls/xtl/include and tpls/xtensor/include. Thanks for clarifying!